src/next-app/pages/quote-request.tsx

```diff
@@ -0,0 +1,328 @@
+            </div>
+            <div className={styles.content}>
+                <div className={styles['text-container']}>
+                    <TextField name="Quote Request Name" placeholder="Enter a quote request name" onChange={setName} required />
+                </div>
+                <div className={styles['text-container']}>
+                    <SingleDropdownField name="Select Vehicle Make" items={makes} onChange={setMake} required />
+                </div>
+                {make === 'Other' ?
+                    <div className={styles['text-container']}>
+                        <div>
+                            <TextField name='Enter Vehicle Make' placeholder="Enter your vehicle make" onChange={setCustomMake} required />
+                        </div>
+                        <div>
+                            <TextField name='Enter Vehicle Model' placeholder="Enter your vehicle model" onChange={setCustomModel} required />
+                        </div>
+                    </div>
+                    : null
+                }
+                {make && make !== 'Other' ?
+                    <div className={styles['text-container']}>
+                        <SingleDropdownField name="Select Vehicle Model" items={models[make]} onChange={setModel} required />
+                    </div>
+                    : null
+                }
+                {model || customModel ?
+                    <div className={styles['text-container']}>
+                        <TextField name="Vehicle Model Year" placeholder="Enter vehicle model year" onChange={setModelYear} required inputType='number' width={'25%'} />
+                    </div> : null
+                }
+            </div>
+            <div className={styles.subtitle}>
+                <label>
+                    Contact Information
+                </label>
+            </div>
+            <div className={styles.content}>
+                <div className={styles['text-container']}>
+                    <TextField name="First Name" placeholder="Enter your first name" onChange={setFirstName} required />
+                </div>
+                <div className={styles['text-container']}>
+                    <TextField name="Last Name" placeholder="Enter your last name" onChange={setLastName} required />
+                </div >
+                <div className={styles['text-container']}>
+                    <TextField name="Phone Number" placeholder="Enter your phone number" onChange={setPhoneNumber} required />
+                </div>
+                <div className={styles['text-container']}>
+                    <TextField name="Email" placeholder="Enter your email" onChange={setEmail} onBlur={handleEmailBlur} errors={emailErrors} required />
+                </div>
+                <div className={styles['text-container']}>
+                    <SingleDropdownField name="Preferred Contact Method" items={['Email', 'Phone']} onChange={setPreferredContact} />

```

Comment:
Letting user type to filter from two items doesn't really make sense. Perhaps we should disable their ability to type in this case and display a placeholder item such as "none" when no choice is selected?